### ISSUE TITLE
smooth_video_progress depends on outdated flutter_hooks

### ISSUE THREAD

**Issue by alguintu** [devkrest/flutter_story_presenter#8]
```
Because every version of flutter_story_presenter depends on smooth_video_progress ^0.0.4 which depends on flutter_hooks ^0.18.4, every version of flutter_story_presenter requires flutter_hooks ^0.18.4.
So, because your_project depends on both flutter_hooks ^0.20.5 and flutter_story_presenter ^0.0.8, version solving failed.
```

The dependency smooth_video_progress hasn't been updated in 2 years and depends on an old version of flutter_hooks, which prevents users of the latest flutter_hooks from using this package. 

Please either fork and update smooth_video_progress's dependency, or include its functionality into the package so it does not cause version solving failure in the future.

I'd be happy to make a PR for this. Thanks.

**Comment by Kaival-Patel** [comment]
@alguintu Thanks for reporting the bug. We will soon look into this. If you have a time, you can do PR, happy to collaborate.

**Comment by Kaival-Patel** [comment]
PR Merged #9
